### PR TITLE
Add postgres

### PR DESCRIPTION
**Pull request by Chris-Hesterman**
Integrated Postgres, front end still plays nice.  Implemented the rest of crud functionality so it works with postman.
Going to restructure backend to conform better with MVC.

### CODE REVIEW

File: server/models/pgmodels.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+  )
+  SELECT * FROM insert1, insert2`;
+
+const deleteReviewQuery = `DELETE FROM reviews WHERE review_id = $1`;
+
+const updateReviewQuery = `UPDATE reviews
+  SET review = $1
+  WHERE review_id = $2
+  RETURNING *`;
+
```
Comment (samasastudio) [inline server/models/pgmodels.js:57]: excellent breaking the queries out this way!  giving me ideas for the future.

File: index.js [hunk 2]
```diff
@@ -118,95 +80,108 @@ server.get('/averageReviews/:itemId', (req, res) => {
...
-    })
-    .catch((err) => {
-      throw err;
-    });
+  pgdb.deleteReview(req.params.reviewId).then((result) => {
+    res.status(200).send(result);
+  });
 });
 
 server.get('/product', (req, res) => {
```
Comment (samasastudio) [inline index.js:143]: don't forget error handling here!

**Review by samasastudio — APPROVED** [review]
heavy refactoring but looks good!  👍

File: index.js (hunk 2 above)
Comment (Chris-Hesterman) [reply index.js:143]: OOPS!  Thanks I'll get on that.

**Pull request merged**